docs/DeveloperGuide.md

```diff
@@ -216,11 +216,11 @@ _Sequence Diagram:_<br/>
 ## Save/Load Feature
 
 The Save/Load feature is implemented by the saveload package.
-At the base of the package, there is the <mark> Saver </mark> 
-and <mark> Loader </mark>  class.
+At the base of the package, there is the `Saver`
+and `Loader` class.
 
-### Architecture
-![Alt text](save_load_feature/Architecture.png)
+### Design
```

Comment:
Consider changing to another header as it might be confusing to readers if there are two design headings(one of which is near the top of the documentation and is used as the heading for the section describing the components).